packages/teams-js/src/internal/handlers.ts

```diff
@@ -154,9 +213,9 @@ export function registerHandlerHelper(
  * @internal
  * Limited to Microsoft-internal use
  */
-export function registerOnThemeChangeHandler(handler: (theme: string) => void): void {
+export function registerOnThemeChangeHandler(apiVersionTag: string, handler: (theme: string) => void): void {
   HandlersPrivate.themeChangeHandler = handler;
-  !isNullOrUndefined(handler) && sendMessageToParent('registerHandler', ['themeChange']);
+  !isNullOrUndefined(handler) && sendMessageToParentWithVersion(apiVersionTag, 'registerHandler', ['themeChange']);
```

Comment:
Why doesn't this hardcode V_2 the way that the other handlers right below this do? In the existing code this handler is only called from one place

Reply:
Yes, there is another call comes from publicAPIs if you take a look on line 112 of publicAPIs.ts file.

Reply:
Makes sense, v_1 is from publicAPI.ts and v_2 is from app.ts. Thanks!